airflow/configuration.py

```diff
@@ -253,6 +253,18 @@ def _validate_config_dependencies(self):
                     + ", ".join(start_method_options)
                 )
 
+        if self.has_option("scheduler", "file_parsing_sort_mode"):
+            list_mode = self.get("scheduler", "file_parsing_sort_mode", fallback="modified_time")
```

Comment:
```suggestion
            list_mode = self.get("scheduler", "file_parsing_sort_mode")
```

We get the fallback for free from `default_airflow.cfg`, no need to duplicate it here.